app/controllers/users_controller.rb

```diff
@@ -1,8 +1,12 @@
 class UsersController < ApplicationController
   def show
-    @github_user = GithubUsers.new
+    @github_user = if current_user.github_token.nil?
```

Comment:
session[:github_token] instead of current_user. I won't make this comment again to save on typing